src/GraphQl/Mutations/OrganizationMutations.ts

```diff
@@ -91,17 +91,6 @@ export const SEND_MESSAGE_TO_CHAT = gql`
       _id
       createdAt
       messageContent
-      replyTo {
-        _id
-        createdAt
-        messageContent
-        sender {
-          _id
-          firstName
-          lastName
-        }
-        updatedAt
-      }
```

Comment:
i was trying to pass the failed tests. so, i did changes thinking it would pass tests.  
[Runs Introspection on the GitHub talawa-api repo on the schema.graphql file](https://github.com/PalisadoesFoundation/talawa-admin/actions/runs/11698785952/job/32579654429?pr=2356#logs)
failed 6 hours ago in 25s